src/shared/components/Settings/Account/MyAccount/index.jsx

```diff
@@ -585,7 +585,7 @@ export default class MyAccount extends React.Component {
                           </div>
                         </div>
                         <div styleName="row">
-                          <div styleName="field col-1 password">
```

Comment:
I remember we've added this class to fix the issue where the label was out of place when the tooltip was shown.

Can you please take a look at that?

Reply:
@ThomasKranitsas sure, looking into it.

Reply:
Hi @ThomasKranitsas, you are right. I missed the purpose the `password` class. I bring it back and removed border & shadow. Can you please check once more. Thanks